knockapi/resources/objects.py

```diff
@@ -111,4 +111,23 @@ def set_channel_data(self, collection, id, channel_id, channel_data):
             dict: Channel data from Knock.
         """
         endpoint = '/objects/{}/{}/channel_data/{}'.format(collection, id, channel_id)
-        return self.client.request('put', endpoint, payload={'data': channel_data})
\ No newline at end of file
+        return self.client.request('put', endpoint, payload={'data': channel_data})
+
+    ##
+    # Messages
+    ##
+
+    def get_messages(self, collection, id, options=None):
```

Comment:
Same question as below. Also in `knockapi/resources/messages.py`.

Reply:
will update!

Reply:
@thomaswhyyou actuall, please see my other comment :)